### ISSUE TITLE
Accessibility of tour "bullets" (aria-current)

### ISSUE THREAD

**Issue by medeaoblongata** [elrumordelaluz/reactour#556]
There's a missed opportunity (and a WCAG violation - Info and Relationships) in the little bullet widget thing that shows the current 'step' in the tour.

Need to add aria-current="true" to the bullet that represents the current element. Attribute should be removed otherwise. 

(And remember that aria attributes work very well in CSS selectors to keep the accessibility tree in sync with the CSS presentation).
